### ISSUE TITLE
getStatements should accept strings for activity and verb parameters

### ISSUE THREAD

**Issue by garemoko** [RusticiSoftware/TinCanJS#128]
When fetching statements it should be possible to pass a string (the activity or verb id) rather than an activity or verb object as the parameter, since that's the final format of the parameter sent. 

see
https://github.com/RusticiSoftware/TinCanJS/blob/master/src/LRS.js#L748-L752

Replace with something like:
```
for (i = 0; i < idProps.length; i += 1) {
    if (typeof cfg.params[idProps[i]] !== "undefined") {
        if (typeof cfg.params[idProps[i]] === "string") {
            params[idProps[i]] = cfg.params[idProps[i]];
        } 
        else
        {
            params[idProps[i]] = cfg.params[idProps[i]].id;
        }
    }
}

```

**Comment by brianjmiller** [comment]
Will go in next release.

**Comment by brianjmiller** [comment]
Available in 0.50.0.
